posydon/visualization/plot_pop.py

```diff
@@ -2,10 +2,10 @@
     "Simone Bavera <Simone.Bavera@unige.ch>",
     "Max Briel <max.briel@unige.ch>",
     ]
-    
+
 import os
 import numpy as np
-import matplotlib.pyplot as plt
```

Comment:
This can't be removed, `plt` is still used at other places.

Reply:
It exists again a few lines below, it was duplicate there, probably by an earlier error

Reply:
I'd probably remove the other instance, to keep the matplotlib imports together.